deploy.py

```diff
@@ -96,13 +96,14 @@ def main():
 
     deploy_parser.add_argument('cluster_name', nargs="?")
     deploy_parser.add_argument('hub_name', nargs="?")
+    deploy_parser.add_argument('--test_notebook')
```

Comment:
How about `--test-notebook` or `--test-ipynb-path` or `--test-notebook-path` instead? I generally like `-` separators than `_` separators for named arguments like this. argparse automatically converts them to `_` so we can access them the same.